build.assets/Makefile

```diff
@@ -115,19 +114,20 @@ build-binaries-fips: buildbox-fips
 # If running in CI and there is no image with the buildbox name:tag combination present locally,
 # the image is pulled from the Docker repository. If this pull fails (i.e. when a new Go runtime is
 # first used), the error is ignored and the buildbox is built using the Dockerfile.
+# BUILDARCH is set explicitly, so it's set with and without BuildKit enabled.
 #
 .PHONY:buildbox
 buildbox:
 	if [[ "$(BUILDBOX_NAME)" == "$(BUILDBOX)" ]]; then \
 		if [[ $${DRONE} == "true" ]] && ! docker inspect --type=image $(BUILDBOX) 2>&1 >/dev/null; then docker pull $(BUILDBOX) || true; fi; \
-		docker build --platform=linux/amd64 \
+		docker build --platform=linux/$(RUNTIME_ARCH) \
```

Comment:
If buildbox always runs with `BUILDARCH`, then maybe we don't need `devbox` at all anymore.

The whole point of `devbox` was to have the protobuf compiler able to run in an arm64 container, but you've fixed that for more than protoc.

Reply:
@zmb3 I was not sure why we have the `devbox` image and what is the further plan for it. If this image was only added to run `protoc` then I can remove as a part of this or other PR.

Reply:
Yep, @xacrimon just recently added the devbox image in the last week or two. I think you're good to remove it if this works out.

Reply:
Can we merge this PR as it is? I'll create another one to remove the `devbox` if this works for everyone.

Reply:
Sure, that works for me.